proxy/coco/src/git_helper.rs

```diff
@@ -22,7 +22,7 @@ pub const GIT_REMOTE_RAD: &str = "git-remote-rad";
 ///   * Could not get the current working directory.
 ///   * Could not create the path to binary directory.
 ///   * Could not copy helper executable to the binary directory.
-pub fn setup(src_dir: &path::PathBuf, dst_dir: &path::PathBuf) -> Result<(), Error> {
+pub fn setup(src_dir: &path::Path, dst_dir: &path::Path) -> Result<(), Error> {
```

Comment:
What's the rationale for this change?

Reply:
I believe `&PathBuf` is equivalent to `&Path`. Analogous to `&Vec<T>` and `&[T]`

Reply:
Yes, clippy now complains about this. (Can’t find the lint name atm.)